src/renderer/utils/store.ts

```diff
@@ -135,12 +145,19 @@ export function unsetDataReducer() {
 }
 
 export function unsetAccountLocalAndStateReducer() {
-  return (state: any, {payload: {account_number: accountNumber}}: PayloadAction<ManagedAccount | ManagedFriend>) => {
+  return (state: any, {payload: {account_number: accountNumber}}: PayloadAction<ManagedAccount>) => {
     delete state[accountNumber];
     localStore.set(getStateName(MANAGED_ACCOUNTS), state);
   };
 }
 
+export function unsetFriendLocalAndStateReducer() {
+  return (state: any, {payload: {friend_number: friendNumber}}: PayloadAction<ManagedFriend>) => {
+    delete state[friendNumber];
+    localStore.set(getStateName(MANAGED_FRIENDS), state);
+  };
+}
+
```

Comment:
once friendNumber becomes accountNumber, this function is no longer needed

Reply:
@angle943 This function is necessary, isn't it?